lib/facebook.js

```diff
@@ -34,19 +34,24 @@ export class Chat {
         return this.send(payload);
     }
 
-    sendButtons(text, buttons) {
-        const payload = {
-            message: {
-                attachment: {
-                    type: 'template',
-                    payload: {
-                        'template_type': 'button',
-                        text: text,
-                        buttons: buttons,
-                    },
+    sendButtons(text, buttons, quickReplies = null) {
```

Comment:
If you make changes to `lib/facebook.js`, you also have to change both the mock in `lib/__mocks__/facebook.js` and the testing library in `lib/testing.js` to conform with the change.

Reply:
Ja, habe mich auch schon gewundert, dass mein Test zuvor durchging,.. :laughing: